src/foam/u2/wizard/data/Canceler.js

```diff
@@ -8,6 +8,7 @@ foam.INTERFACE({
   package: 'foam.u2.wizard.data',
   name: 'Canceler',
```

Comment:
```suggestion
  name: 'Canceller',
```

Reply:
Google said it was just one L. Although I kinda wish it was `SaveEr` and `CancelEr` 'cause they're all `Er`s (things that do things)